### ISSUE TITLE
Review recycler issue inside the pager

### ISSUE THREAD

**Issue by troZee** [callstack/react-native-pager-view#819]
There might be an issue inside the pagerview. Please checkout the below link: 

https://github.com/facebook/react-native/issues/42732

**Comment by cipolleschi** [comment]
Yep, the problem here is that pager-view is using a view provided by fabric as content-view. That should not be done as UIPageViewController modifies the content view and, if then Fabric recycle it, it might serve a modified view to some other surface.

Instead of `init` the pager view with a React Native view, please consider adding the React Native view as subview of the UIKit view, so that we do not risk to recycle the UIKit view.

I can share more details if needed!

**Comment by troZee** [comment]
Thank you @cipolleschi and @j-piasecki for checking this issue 🙏 

> I can share more details if needed!

So I have a question to clarify that. Do you mean to wrap every react native children view with UIView, so UIPageViewController will not interfere with react native children view?
